junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/config/DisplayNameGeneratorParameterConverter.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.engine.config;
+
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.platform.commons.logging.Logger;
+import org.junit.platform.commons.logging.LoggerFactory;
+import org.junit.platform.commons.util.ReflectionUtils;
+import org.junit.platform.engine.ConfigurationParameters;
+
+/**
+ * @since 5.5
+ */
+class DisplayNameGeneratorParameterConverter {
+
+	private static final Logger logger = LoggerFactory.getLogger(DisplayNameGeneratorParameterConverter.class);
+
+	DisplayNameGenerator get(ConfigurationParameters configurationParameters, String key) {
+		return configurationParameters.get(key).map(String::trim).filter(
```

Comment:
Please either use trailing `//` comments or `// @formatter:off -- // @formatter:on` blocks to make this conform to the formatting style used elsewhere in the code base.

Reply:
👍